src/main/java/com/rcv/Main.java

```diff
@@ -70,41 +70,18 @@ public static void main(String[] args) {
   // returns: the new ElectionConfig object, or null if there was a problem
   static ElectionConfig loadElectionConfig(String configPath) {
     // config: the new object
-    ElectionConfig config = null;
-
-    // tracks whether we encountered any critical file system errors
-    boolean encounteredFileError = false;
+    ElectionConfig config;
 
     // rawConfig holds the basic election config data parsed from json
     RawElectionConfig rawConfig =
         JsonParser.parseObjectFromFile(configPath, RawElectionConfig.class);
 
     if (rawConfig == null) {
-      System.err.println(String.format("Failed to load config file: %s", configPath));
+      Logger.executionLog(Level.SEVERE, String.format("Failed to load config file: %s", configPath));
+      return null;
```

Comment:
We're not supposed to have multiple returns, ya jabroni!

Reply:
![28452_the-rock_l](https://user-images.githubusercontent.com/838732/44056182-50e12f28-9efc-11e8-8619-285560504406.jpg)

Reply:
This man could easily be our next president if we get more RCV laws passed.